libunwind/src/UnwindCursor.hpp

```diff
@@ -2064,6 +2077,51 @@ bool UnwindCursor<A, R>::getInfoFromSEH(pint_t pc) {
       }
     }
   }
+#elif defined(_LIBUNWIND_TARGET_ARM)
```

Comment:
Given how similar the code is to the aarch64 version, it's a bit unfortunate to have to duplicate all of it. Maybe we could condition `UNWIND_INFO_ARM` on the target and reuse it for both 64-bit and 32-bit code? I think that would reduce the differences to just the `end_ip` calculation.